corehq/apps/app_manager/models.py

```diff
@@ -4074,13 +4075,13 @@ def form_record_url(self):
         return '/a/%s/api/custom/pact_formdata/v1/' % self.domain
 
     @absolute_url_property
-    def hq_profile_url(self):
+    def profile_url(self):
         return "%s?latest=true" % (
             reverse('download_profile', args=[self.domain, self._id])
         )
 
     @absolute_url_property
-    def hq_media_profile_url(self):
```

Comment:
I think your original intention was to support different URL's for remote apps but not 100% sure:
* https://github.com/dimagi/commcare-hq/commit/0a513fb5f23b1929e136597fcfb3545649c821fa

After that I think the pattern was just copied:
* https://github.com/dimagi/commcare-hq/commit/f77abe8a49d5dd55adf69667b6a50db0bb43e226